_episodes/02-starting-with-data.md

```diff
@@ -329,14 +329,14 @@ with `surveys_df.columns`.
 
 Methods are called in a similar fashion using the syntax `df_object.method()`.
 As an example, `surveys_df.head()` gets the first few rows in the DataFrame
-`surveys_df` using **the `head()` method**. With a method, we can supply extra
+`surveys_df` using the `head()` **method**. With a method we can supply extra
```

Comment:
```suggestion
`surveys_df` using the `head()` method. With a method we can supply extra
```

I think this should not be in bold, if you are already emphasising the words "methods" and "attributes" above.